### ISSUE TITLE
ecmaFeatures appears to be ignored

### ISSUE THREAD

**Issue by liammagee** [adametry/gulp-eslint#73]
When I try to set 'ecmaFeatures' explicitly, the option doesn't seem to pass through:

    .pipe(eslint(
        {
            ecmaFeatures: { templateStrings: true },
            // envs: [ 'es6' ]
        }
    ))

Result:

  5524:21  error  Unexpected token ILLEGAL

Removing the comment doesn't produce this error. Happy to do a pull request if this is actually an issue.

**Comment by lucasmotta** [comment]
I'm having the same issue here.

**Comment by adametry** [comment]
It's true that eslint's Config doesn't include ecmaFeatures directly; Though, it is possible to set ecmaFeatures in it's baseConfig.
```
.pipe(eslint({
	baseConfig: {
		ecmaFeatures: {
			templateStrings: true
		}
	}
}))
```

@lucasmotta, are you using the latest gulp-eslint? If so, there might be more environmental influences. Can you provide a example so I can reproduce the issue?

**Comment by adametry** [comment]
A release candidate has been published that will automatically migrate "ecmaFeatures" to baseConfig. Give it a try and let me know how it works.
```shell
npm install gulp-eslint@rc
```

**Comment by adametry** [comment]
gulp-eslint@1.1.0 has been released. I'll close this bug since a fix has been released. If you run into further problems, let me know.
